app/mm-redux/selectors/entities/groups.ts

```diff
@@ -45,9 +55,67 @@ export function getGroupMembers(state: GlobalState, id: string) {
     return groupMemberData.members;
 }
 
-const teamGroupIDs = (state: GlobalState, teamID: string) => (state.entities.teams.groupsAssociatedToTeam[teamID] == null ? undefined : state.entities.teams.groupsAssociatedToTeam[teamID].ids == null ? undefined : state.entities.teams.groupsAssociatedToTeam[teamID].ids) || [];
+export function searchAssociatedGroupsForReferenceLocal(state: GlobalState, term: string, teamId: string, channelId: string): Array<Group> {
+    // uncomment when USE_GROUP_MENTIONS is checked-in
+    // if (!haveIChannelPermission(state, {
+    //     permission: Permissions.USE_GROUP_MENTIONS,
+    //     channel: channelId,
+    //     team: teamId,
+    // })) {
+    //     return emptyList;
+    // }
+
+    const groups = getAssociatedGroupsForReference(state, teamId, channelId);
+    if (!groups) {
+        return emptyList;
+    }
+    const filteredGroups = filterGroupsMatchingTerm(groups, term);
+    return filteredGroups.sort((groupA: Group, groupB: Group) => groupA.name.localeCompare(groupB.name));
```

Comment:
You are performing a localeCompare without passing a locale as an argument, does this need to be localeCompare??

Reply:
Passing in the locale now. Done!